app/controllers/api/v1/authentification_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::AuthenticationController < ApiController
+  skip_before_action :authenticate_token!
+
+  def create
+    user = User.find_by(username: user_params[:username])
+
+    if user && user.valid_password?(user_params[:password])
+      render json: { token: JsonWebToken.encode(sub: user.id) }
+    else
+      render json: { errors: ['Invalid username or password'] }, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:username, :password)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] It might be a typo or not however, I realized that the file name was named as "authentification" but the controller is "authentication". Might this be a typo? I highly suggest that you have a consistent naming structure for your controllers and the file names as well.